### ISSUE TITLE
关于论文中before generating precise ground truth Gaussian peaks

### ISSUE THREAD

**Issue by Sunstin** [hellojialee/Improved-Body-Parts#23]
Hi,jialee
有幸阅读到simple pose这篇论文，关于论文当中产生GT高斯热图的解释我没有理解为何要涉及到R(stride步长)以及对位置的换算。论文当中是这样描述的：
By the way, we map the pixel p at the location p(x; y) in
the j-th ground truth heatmap to its original floating point
location p~(~ x; y~) = p~(x · R + R=2 − 0:5; y · R + R=2 − 0:5)
in the input image, in which R is the output stride, before
generating the precise ground truth Gaussian peaks.

在heatmapper.py中有看到相关注释，比如：
x,y coordinates of centers of bigger grid, stride / 2 -0.5是为了在计算响应图时，使用grid的中心
basically we should use center of grid, but in this place classic implementation uses left-top point.
期待您的回复，谢谢~

**Comment by Sunstin** [comment]
在知乎看到关于这个问题的解答，应该是你们分享的，感谢~
